noir-projects/noir-protocol-circuits/crates/rollup-lib/src/block_merge/block_merge_rollup_inputs.nr

```diff
@@ -185,7 +185,7 @@ mod tests {
     fn valid_previous_circuit_block_root() {
         let mut inputs = default_block_merge_rollup_inputs();
         let vk_tree: MerkleTree<fixtures::vk_tree::VK_TREE_WIDTH> =
-            comptime { fixtures::vk_tree::get_vk_merkle_tree() };
+            comptime { fixtures::vk_tree::VK_MERKLE_TREE };
```

Comment:
We can remove all these comptime blocks at the same time.

Reply:
Done!